Pod/Classes/Categories/PINImage+ScaledImage.m

```diff
@@ -0,0 +1,53 @@
+//
+//  UIImage+ScaledImage.m
+//  Pods
+//
+//  Created by Michael Schneider on 2/9/17.
+//
+//
+
+#import "PINImage+ScaledImage.h"
+
+inline PINImage *PINScaledImageForKey(NSString * __nullable key, PINImage * __nullable image) {
```

Comment:
Thanks! We had the function as export as well as I used it initially only one time. Removed the export for now and made it static we don't use it outside.